torch/testing/_internal/common_methods_invocations.py

```diff
@@ -214,38 +231,40 @@ def __init__(self,
                  handles_large_floats=True,  # whether the op correctly handles large float values (like 1e20)
                  handles_extremals=True,  # whether the op correctly handles extremal values (like inf)
                  handles_complex_extremals=True,  # whether the op correct handles complex extremals (like inf -infj)
-                 promotes_integers_to_float=False,  # whether op promotes unary output to float or not
```

Comment:
@Lilyjjo the argument `promotes_integers_to_float` was removed here, but is used below in `asin`, `asinh`, `atan` `UnaryUfuncInfo` definitions, is it intentional too ? Thanks.
Context: I'm failing to resolve conflict with one of my branches while adding sparse support for unary func tests.